components/TableOfContents/TableOfContents.jsx

```diff
@@ -59,9 +48,7 @@ function TableOfContents({ tableOfContentArr, currentPageUrl = [], anchorLinks,
     )
 
     useEffect(() => {
-        const arrayWithAnchorElements = Array.from(
-            document.querySelectorAll('h1[id], h2[id], h3[id]')
-        )
+        const arrayWithAnchorElements = Array.from(document.querySelectorAll('h1[id], h2[id]'))
```

Comment:
nit:

Let's replace `Array.from` to `[... document.querySelectorAll('h1[id], h2[id]')]`